node_modules/col-core/lib/storage.js

```diff
@@ -176,9 +223,6 @@ var putObjectToS3 = function(key, filePath, callback) {
       return callback(err);
     }
 
-    // Generate a signed download URL
-    var expires = 2 * 365 * 24 * 60 * 60;
-
```

Comment:
Strangely, we have here an unused cut-and-paste from the preview-service code that started the whole problem.